src/chrome/content/rules/NetCologne.xml

```diff
@@ -1,60 +1,90 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://webtv.netcologne.de/ => https://webtv.netcologne.de/: (6, 'Could not resolve host: webtv.netcologne.de')
-
-	NetCologne Gesellschaft für Telekommunikation mbH
+<!--NetCologne Gesellschaft für Telekommunikation mbH
 
 	Other NetCologne rulesets:
 
 		- DotCologne.de.xml
 
-	Insecure cookies are set for these hosts:
+	Cert expired:
+		- gk-shop
+		- hosting
+
+	Cert mismatch:
+		- ifr
+		- mg
+		- microsoft-hosting
+		- mirror2
+		- navigator
+		- partner
+		- prtg
+		- reg
+		- report
+		- vcp
+		- webcam
+		- webcluster01
+
+	Chain issues:
+		- cpbxlogin
+		- in-service
+		- ntmail
+		- owa
+		- sso-test
+		- vsm
+
+	Self-signed cert:
+		- cftp
+		- kftp
 
-		- tracking.netcologne.de
+	TLS error:
+		- deny
 
 -->
-<ruleset name="NetCologne.de (partial)" default_off='failed ruleset test'>
+<ruleset name="NetCologne.de (partial)">
 
 	<!--	Direct rewrites:
 				-->
-	<target host="aktion.netcologne.de"/>
-	<target host="comcenter.netcologne.de"/>
-	<target host="debian.netcologne.de"/>
-	<target host="mirror.netcologne.de"/>
-	<target host="mobilshop.netcologne.de"/>
-	<target host="musicbox.netcologne.de"/>
-	<target host="netcologne.de"/>
-	<target host="netkompakt-fuer-mitarbeiter.netcologne.de"/>
-	<target host="onlineservice.netcologne.de"/>
-	<target host="planauskunft.netcologne.de"/>
-	<target host="shop.netcologne.de"/>
-	<target host="statistik.netcologne.de"/>
-	<target host="tracking.netcologne.de"/>
-	<target host="webtv.netcologne.de"/>
-	<target host="www.netcologne.de"/>
```

Comment:
The following hosts have been removed:
```
netcologne.de
www.netcologne.de
aktion.netcologne.de
```
Please keep the rule for them or add a comment why you removed them.